runner/actions-runner-dind-rootless.ubuntu-20.04.dockerfile

```diff
@@ -5,7 +5,7 @@ ARG RUNNER_VERSION
 ARG RUNNER_CONTAINER_HOOKS_VERSION=0.2.0
 # Docker and Docker Compose arguments
 ENV CHANNEL=stable
-ARG DOCKER_COMPOSE_VERSION=v2.6.0
+ARG DOCKER_COMPOSE_VERSION=v2.16.0
```

Comment:
Note: This does correspond to what is used in GitHub-hosted Ubuntu 20.04 runners: https://github.com/actions/runner-images/blame/bcb2886e5b0fb66c491f66640461128755c06885/images/linux/Ubuntu2004-Readme.md?plain=1#L80